ArgoProject/src/systems/FollowEntitySystem.cpp

```diff
@@ -0,0 +1,26 @@
+﻿#include "stdafx.h"
+#include "FollowEntitySystem.h"
+#include "components/FollowEntity.h"
+#include "components/Location.h"
+#include "components/Input.h"
+
+void app::sys::FollowEntitySystem::update(app::time::seconds const & dt)
+{
+	m_registry.view<comp::FollowEntity, comp::Location>()
+		.each([&, this](app::Entity const entity, comp::FollowEntity & _followEntity, comp::Location & _location)
+	{
+		if (m_registry.valid(_followEntity.entity) && m_registry.has<comp::Input>(_followEntity.entity))
```

Comment:
use `m_registry.view` and `inputView.contains` here instead of `m_registry.has` function.